EasyPost/Tracker.cs

```diff
@@ -102,31 +103,30 @@ public static TrackerCollection All(Dictionary<string, object> parameters = null
         /// </summary>
         /// <param name="id">String representing a Tracker. Starts with "trk_".</param>
         /// <returns>EasyPost.Tracker instance.</returns>
-        public static Tracker Retrieve(string id)
+        public static async Task<Tracker> Retrieve(string id)
         {
             Request request = new Request("trackers/{id}");
             request.AddUrlSegment("id", id);
 
-            return request.Execute<Tracker>();
+            return await request.Execute<Tracker>();
         }
 
         /// <summary>
         ///     Create a list of trackers
         /// </summary>
         /// <param name="parameters">A dictionary of tracking codes and carriers</param>
         /// <returns>True</returns>
-        public static bool CreateList(Dictionary<string, object> parameters)
+        public static async Task<bool> CreateList(Dictionary<string, object> parameters)
         {
-            Request request = new Request("trackers/create_list", RestSharp.Method.Post);
+            Request request = new Request("trackers/create_list", Method.Post);
             request.AddBody(new Dictionary<string, object>
             {
                 {
                     "trackers", parameters
                 }
             });
-            request.Execute<Tracker>();
+            return await request.Execute();
```

Comment:
Quick question, should this be like `return await request.Execute<Tracker>();`?

Reply:
No, this API endpoint doesn't return any data. `request.Execute()` returns true/false based on whether the API returned an error status code, while `request.Execute<T>()` returns a T-type object.